### ISSUE TITLE
fdk-aac-encoder: rename to fdkaac, fdk_aac: rename to libfdk_aac

### ISSUE THREAD

**Issue by milahu** [milahu/nur-packages#4]
fdk-aac-encoder: rename to fdkaac
commit 336a7bde1015007bcec75dcf9ee25bd6cbed51ad

```diff
--- a/default.nix
+++ b/default.nix
+  # this packages is called "fdkaac" on github, debian, archlinux, ...
+  # the library is called "fdk_aac", so better rename that to "libfdk_aac"
+  fdkaac = pkgs.fdk-aac-encoder;
```

fdk_aac: rename to libfdk_aac or libfdk-aac
because it is a "library only" packages = it has no executable files
in debian, this package is [libfdk-aac-dev](https://packages.debian.org/search?suite=bookworm&section=all&arch=any&searchon=names&keywords=libfdk-aac)
in archlinux,this package is [libfdk-aac](https://archlinux.org/packages/extra/x86_64/libfdk-aac/)
in ffmpeg, this codec is [libfdk_aac](https://trac.ffmpeg.org/wiki/Encode/AAC)

ping maintainer of fdk-aac-encoder: @lunik1 

ping maintainer of fdk_aac: @codyopel

**Comment by lunik1** [comment]
I think you opened this on the wrong repo

**Comment by milahu** [comment]
nah, [im banned from nixos](https://github.com/milahu/hate-maintainers-censored) for "hate speech", so im talking to myself here
